app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+# Returns the full title on a per-page basis.
+def full_title(page_title = '')
+base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
```suggestion
  base_title = "Ruby on Rails Tutorial Sample App"
```
check lại indent cho file này nhé
dòng con thụt vào so với thằng cha 1 tab